convert2rhel/toolopts.py

```diff
@@ -43,7 +45,9 @@ def __init__(self):
         self.enablerepo = []
         self.disablerepo = []
         self.pool = None
-        self.serverurl = None
```

Comment:
There are some unit tests working with the toolopts.serverurl, like `test_custom_serverurl`.

Reply:
unit tests should now be updated to stop using serverurl